maven-3.5-jdk8-alpine/Docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM maven:3.5-jdk-8-alpine as build
+
+RUN apk update && \
+  apk add bash
+
+ADD configure_user_and_group.bash entrypoint.bash /opt/
+ADD settings.xml /mnt/settings.xml
+
+WORKDIR /opt/maven
+
+RUN chmod u+x /opt/configure_user_and_group.bash
```

Comment:
Since this script is sourced, not executed, there is no need to give executable permission on it. This line should be removed.

Reply:
Removed